presto-main/src/main/java/com/facebook/presto/sql/gen/PageFunctionCompiler.java

```diff
@@ -423,7 +424,8 @@ private static MethodDefinition generateProcessMethod(
         return method;
     }
 
-    private List<MethodDefinition> generateCommonSubExpressionMethods(
+    private static List<MethodDefinition> generateCommonSubExpressionMethods(
+            Metadata metadata,
```

Comment:
Is this needed?

Reply:
I think this change can be removed, right?

Reply:
@rongrong it is removed already